src/quest_manager/models.py

```
@@ -439,13 +456,39 @@ def get_available_without_course(self, user):
+
+        Args:
+            user: The user requesting archived quests
+
+        Returns:
+            QuerySet of archived quests if user is staff, empty list otherwise
+        """
+        if user.is_staff:
+            qs = self.get_queryset().filter(archived=True)
+            return qs
+        else:
+            return qs.none()
```

Pylint (3.3.7)

[refactor] Unnecessary "else" after "return", remove the "else" and de-indent the code inside it

(R1705)